app/PostStatus.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class PostStatus extends Model
+{
+    protected $table = 'post_statuses';
```

Comment:
If you follow Laravel naming convention then you do not need to specify table name. Do it only if it differs from convention. In all other cases Laravel calculates it dynamically.
https://laravel.com/docs/5.8/eloquent#eloquent-model-conventions

Reply:
Didn't know that PostStatus will be assigned to the 'post_statuses' table. Didn't see that in their documentation.
Fixed.

Reply:
Definetaly it's there _)

![image](https://user-images.githubusercontent.com/18187060/70307198-4cc96880-1811-11ea-8542-1af3b7b5ebfc.png)